app/src/main/res/values/strings.xml

```diff
@@ -28,7 +28,8 @@
   <string name="enter_password_activity_unlock_button">Unlock Wallet</string>
   <string name="enter_password_activity_creating_wallet_progress_dialog">Creating Wallet…</string>
   <string name="enter_password_activity_unlocking_wallet_progress_dialog">Unlocking Wallet…</string>
-  <string name="enter_password_activity_xml_universe">Connecting to universe&#160;</string>
+  <string name="enter_password_activity_xml_universe">Connecting to universe\u0020</string>
+  <string name="enter_password_activity_xml_specific_node">and node\u0020</string>
```

Comment:
Don't change this now, but for future reference, maybe we could add a short comment clarifying that `\u0020` is the unicode character for a "single breakable space".

Also, is a breakable space we want here (`\u0020`) or the non-breakable counterpart (`\u00A0`). Depends on what we're concatenating with I guess.

Reply:
agree